src/state/apis/zapper/zapperApi.ts

```diff
@@ -172,6 +161,9 @@ export const zapperApi = createApi({
         let data: V2NftUserItem[] = []
 
         const userAddresses = accountIdsToEvmAddresses(accountIds)
+        // TODO: support other networks - note, Polygon is officially supported but returns no data
+        // So we'll need to use another provider than Zapper, and conform to the same interface
+        const network = chainIdToZapperNetwork(ethChainId)!
```

Comment:
Possibly a regression as far the NFTs being displayed go, though it is a correctness improvement: we never had explicit support for NFTs on other chains (i.e floor price is denoted in ETH, Opensea link isn't guaranteed to work etc).
Do we want to keep the current implementation with no filter, which gives us all networks, or narrow it down to the one we know works as intended?

Reply:
Confirmed this is a regression - let's keep the previous implementation, this will cost more points for networks Zapper doesn't actually supports as advertised, but this is the only way unfortunately

Reply:
38ea8bb747b0af767dd8bef3df072e04527d58ac